sdk/runtime/metrics.go

```diff
@@ -10,9 +10,50 @@ import (
 	"strconv"
 	"time"
 
+	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
+func NewCounter(runenv *RunEnv, name string, help string) prometheus.Counter {
+	c := prometheus.NewCounter(prometheus.CounterOpts{
+		Name: name,
+		Help: help,
+	})
+	runenv.MetricsPusher.Collector(c)
+	return c
+}
+
+func NewGauge(runenv *RunEnv, name string, help string) prometheus.Gauge {
+	g := prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: name,
+		Help: help,
+	})
+	runenv.MetricsPusher.Collector(g)
+	return g
+}
+
+func NewHistogram(runenv *RunEnv, name string, help string) prometheus.Histogram {
+	h := prometheus.NewHistogram(prometheus.HistogramOpts{
```

Comment:
I think we are supposed to define the buckets here, as per https://godoc.org/github.com/prometheus/client_golang/prometheus#Histogram ?

Reply:
You're right, fixed!